overlord/snapstate/handlers.go

```diff
@@ -1355,6 +1355,28 @@ func (m *SnapManager) restoreUnlinkOnError(t *state.Task, info *snap.Info, tm ti
 	return err
 }
 
+func onRefreshInhibitionTimeout(t *state.Task, snapName string) error {
+	chg := t.Change()
+
```

Comment:
nitpick: we can just take a change here
```suggestion
func onRefreshInhibitionTimeout(chg *state.Change, snapName string) error {
```